stories/flow.stories.js

```diff
@@ -32,3 +35,15 @@ export const Sample3Dagre = () => (
 export const Sample4Dagre = () => (
   <Flow data={sample4} config={{ layout: 'dagre' }} />
 )
+
+export const InSUIModal = () => (
+  <Modal open>
+    <Modal.Content>
+      <Card fluid>
+        <Card.Content style={{ height: 300 }}>
```

Comment:
I noticed that The "overflow" of Panel happens due to manually declared `height` value. In other words, if passing this value down to `<ReactFlow>` through `<Layout>` would solve the issue.

Reply:
Also `padding: '0em'` would get rid of the spacing around the panel.

Reply:
> I noticed that The "overflow" of Panel happens due to manually declared `height` value. In other words, if passing this value down to `<ReactFlow>` through `<Layout>` would solve the issue.

do you mean to not confine the height? in that case, how do we dictate how large the canvas should be?